src/datamigration/azext_datamigration/manual/custom.py

```diff
@@ -93,213 +55,121 @@ def datamigration_assessment(connection_string=None,
 
 
 # -----------------------------------------------------------------------------------------------------------------
-# Assessment helper function to test whether the given cofig_file_path is valid and has valid action specified.
+# Performance Data Collection Command Implementation.
 # -----------------------------------------------------------------------------------------------------------------
-def validate_config_file_path(path):
-
-    if not os.path.exists(path):
-        raise InvalidArgumentValueError(f'Invalid config file path: {path}. Please provide a valid config file path.')
+def datamigration_performance_data_collection(connection_string=None,
+                                              output_folder=None,
+                                              perf_query_interval=None,
+                                              static_query_interval=None,
+                                              number_of_interation=None,
+                                              config_file_path=None):
```

Comment:
You can set param default vaules here, for example, `static_query_interval=3600`. And the help message will be auto-appended saying "Default: 3600"

Reply:
Done